components/brave_rewards/resources/tip/reducers/tip_reducer.ts

```diff
@@ -97,6 +100,16 @@ const publishersReducer: Reducer<RewardsTip.State> = (state: RewardsTip.State =
       state.reconcileStamp = action.payload.stamp
       break
     }
+    case types.GET_BALANCE:
+      chrome.send('brave_rewards_tip.fetchBalance')
+      break
+    case types.ON_BALANCE: {
+      state = { ...state }
+      if (payload.status !== 1) {
```

Comment:
Seems unclear what a status of 1 represents. Is there any way to make it an enum? Or at least a comment here to explain perhaps.

Reply:
done